src/ServiceControlInstaller.PowerShell/ServiceControlInstaller.Powershell.dll-help.xml

```diff
@@ -1271,13 +1484,18 @@ Get-SecurityIdentifier MyDomain\Foo</dev:code>
 			<command:parameter required="false" variableLength="false" globbing="false" pipelineInput="false" position="2">
 				<maml:name>Password</maml:name>
 				<maml:description>
-					<maml:para>
-            The Password for the Service Account.  When no account is specified then LocalSystem is used.
-            LocalSystem does not require a password. For Active Directory Group Managed Service Accounts no password should be specified.
-          </maml:para>
+					<maml:para>The Password for the Service Account.  When no account is specified then LocalSystem is used.
+LocalSystem does not require a password. For Active Directory Group Managed Service Accounts no password should be specified.</maml:para>
 				</maml:description>
 				<command:parameterValue required="true" variableLength="false">String</command:parameterValue>
 			</command:parameter>
+			<command:parameter required="false" variableLength="false" globbing="false" pipelineInput="false" position="named">
+				<maml:name>Force</maml:name>
+				<maml:description>
+					<maml:para>Force the installation even if the directories specified are not empty </maml:para>
```

Comment:
Nothing but not recommended which is why the force is there.  You'd use this if you wanted to create a service and re-attach an existing DB.